test/e2e/tsdx-build-default.test.ts

```diff
@@ -105,6 +105,15 @@ describe('tsdx build :: zero-config defaults', () => {
     shell.mv('package-og.json', 'package.json');
   });
 
+  it('should replace lodash/fp correctly', () => {
+    const output = execWithCache('node ../dist/index.js build');
+    expect(output.code).toBe(0);
+
+    // ensures correct lodash/fp import was used
+    const matched = grep(/lodash\/fp/, ['dist/build-default.*.js']);
+    expect(matched).toBeTruthy();
+  });
+
```

Comment:
Could you move this test above the `clean` test, but below the `regeneratorRuntime` test? Basically 1 up. The clean test is a cache buster, so I'd prefer for that to stay at the end.